wasm/wasm_source/src/vp_implicit.rs

```diff
@@ -710,7 +710,8 @@ mod tests {
             std::fs::read(VP_ALWAYS_TRUE_WASM).expect("cannot load wasm");
 
         // hardcoded hash of VP_ALWAYS_TRUE_WASM
-        tx_env.init_parameters(None, None, Some(vec!["E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855".to_string()]));
+        // this must be lower case, as its written into genesis as lower case
+        tx_env.init_parameters(None, None, Some(vec!["e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855".to_string()]));
```

Comment:
I don't think these changes are necessary as the `init_paramaters` calls to the params update functions that apply lowercasing now